src/transport/abstract-transport.js

```diff
@@ -20,6 +20,7 @@ function AbstractTransport(address, timeout, messageHandler) {
   this.awaitingPump = false;
   this.connected = false;
   this.backoffMilliseconds = 1;
+  this.nextReconnectListenerToken = 1;
```

Comment:
Ah, alright, so this is a completely different token than the one tracked by ethrpc-transport-adapter.  Does this question make more sense applied to the `nextToken` variable in ethrpc-transport-adapter?